pages/pools/[[...pool]].tsx

```diff
@@ -0,0 +1,143 @@
+import StatBox from 'components/StatBox';
+import { BigNumber, utils } from 'ethers';
+import Head from 'next/head';
+import { useEffect, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { useRecoilValue } from 'recoil';
+import { useGetUniswapStakingRewardsAPY } from 'sections/pool/useGetUniswapStakingRewardsAPY';
+import PoolTabs from 'sections/pool/TabsButton';
+import { useGUNILPToken } from 'sections/pool/useGUNILPToken';
+import { isL2State, walletAddressState } from 'store/wallet';
+import styled from 'styled-components';
+import media from 'styled-media-query';
+import { ExternalLink, FlexDivCol, LineSpacer, StatsSection } from 'styles/common';
+import { WETHSNXLPTokenContract } from 'constants/gelato';
+import { InfoContainer, Subtitle } from 'sections/layer2/components/common';
+import synthetix, { NetworkId } from '@synthetixio/contracts-interface';
+
+function Pool() {
+	const [LPBalance, setLPBalance] = useState(BigNumber.from(0));
+	const [rewardsToClaim, setRewardsToClaim] = useState(BigNumber.from(0));
+	const [allowanceAmount, setAllowanceAmount] = useState(BigNumber.from(0));
+	const [stakedTokens, setStakedTokens] = useState(BigNumber.from(0));
+	const { t } = useTranslation();
+	const walletAddress = useRecoilValue(walletAddressState);
+	const isL2 = useRecoilValue(isL2State);
+	const snx = synthetix({ networkId: NetworkId['Mainnet-Ovm'], useOvm: true });
+	const { balanceOf, rewards, allowance, approve, stakedTokensBalance } = useGUNILPToken({
+		pool: 'weth-snx',
+		userAddress: walletAddress,
+	});
+	const rates = useGetUniswapStakingRewardsAPY({
+		stakingRewardsContract: snx.contracts.StakingRewardsSNXWETHUniswapV3,
+		tokenContract: WETHSNXLPTokenContract,
+	});
+
+	useEffect(() => {
+		fetchBalances();
```

Comment:
To make lint happy we should put fetchBalances in a useCallback and have fetch balances as dependency

Reply:
Fixed